plugins/index.js

```diff
@@ -16,24 +14,15 @@ class AppiumInspectorPlugin extends BasePlugin {
 
   static async openInspector(req, res) {
     const reqPath = req.path === PLUGIN_ROOT_PATH ? INDEX_HTML : req.path.substring(10);
-    res.sendFile(reqPath, {root: `${dirname(fileURLToPath(import.meta.url))}/${DIST_DIR}`});
+    res.sendFile(reqPath, {root: ROOT_DIR});
   }
 
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
   static async updateServer(expressApp, httpServer, cliArgs) {
-    const staticFilesRouter = express.Router();
-    const PUBLIC_DICRECTORY = getPublicDirectory();
-    staticFilesRouter.use(express.static(PUBLIC_DICRECTORY));
-    expressApp.use(staticFilesRouter);
```

Comment:
@KazuCocoa This was added coz the assets folder was not served. Are you able to get the inspector UI without this code?

Reply:
I guess you didn't set `--base /inspector` to build the base browser package. Then, inspector contents refer to `/` as the root. So, you could see requests to `/` without `/inspector` by inspector contents in appium server logs